.github/workflows/deploy-personal-dev-environment.yml

```diff
@@ -63,30 +69,126 @@ jobs:
 
       - uses: ./.github/actions/setup-zsh
 
-      - name: Checkout all repos
-        run: |
-          source uhd.sh
-          uhd gh clone
-        shell: zsh {0}
+      - name: Checkout frontend repo
+        uses: actions/checkout@v4
+        with:
+          repository: UKHSA-Internal/data-dashboard-frontend
+          path: data-dashboard-frontend
+          ref: ${{ inputs.frontend_branch }}
+
+      - name: Checkout infra repo
+        uses: actions/checkout@v4
+        with:
+          path: data-dashboard-infra
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v3
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
```

Comment:
We need QEMU and docker buildx so that we can use the emulator when building the  image